articles/10-tips-for-working-with-node.js/index.md

```diff
@@ -0,0 +1,295 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/10-tips-working-with-nodejs/
+title: 10 Tips for Working with Node.js	
+description: Discover ten of the common tips you need to know from	
+getting started to deploying your first Node.js application.	
+author: joseph-chege
+date: 2020-08-22T00:00:00-10:00	
+topics: [Node.js]	
+excerpt_separator: <!--more-->	
+images:		
+  - url: /engineering-education/10-tips-for-working-with-nodejs/hero.jpg	
+    alt: laptop node.js website
+---
+As a developer getting started with Node.js application development, there a few basic skills that you need to be familiar with as you start to build your first Node.js application.
+
+This article will cover some of the common tips you need to know in order to have a strong background to get started with Node.js application development.
+
+## 1. Understand the Node.js Package Manager (NPM)
+
+NPM is an easy to use package manager for the Node.js platform. It helps to install packages and manage versions and dependencies. NPM helps to install the package you want with just a single command. 
+
+To use NPM, you need Node.js to be installed on your computer. For OS X and Windows, the Node.js installer has an easy to use interface and NPM is included. Developers using Linux need to run the following command in their package manager to set up Node.js and NPM.
+
+```bash
+sudo apt-get update
+
+sudo apt-get install nodejs
+
+sudo apt-get install npm
+```
+
+You can then verify that NPM has been installed by writing the following in your
```

Comment:
You could also say that running `node -v` will detect the version of Node.js they have just installed since there are different version requirements for packages.